### PR TITLE
Changes .gitattributes to be more friendly for web projects

### PR DESCRIPTION
**Pull request by ysims**
This pull request fixes issues caused by the '.gitattributes' file. 

Files that are not text files (binary) will no longer have characters assumed to be CR-LF (0d0a) turned into LF-only (0a).

### CODE REVIEW

File: .gitattributes [hunk 1]
```diff
@@ -1 +1,27 @@
-* text=auto eol=lf
\ No newline at end of file
+## GITATTRIBUTES FOR WEB PROJECTS
```
Comment (JosephusPaye) [inline .gitattributes]: This header comment is unnecessary IMO.

File: .gitattributes [hunk 2]
```diff
@@ -1 +1,27 @@
...
+#   binary  These files are binary and should be left untouched.
+#
+# Note that binary is a macro for -text -diff.
+######################################################################
+
+# Auto detect
+##   Handle line endings automatically for files detected as
+##   text and leave all files detected as binary untouched.
+##   This will handle all files NOT defined below.
+*                 text=auto
```
Comment (JosephusPaye) [inline .gitattributes]: I think we're missing the `eol=lf` option

File: .gitattributes (hunk 1 above)
Comment (ysims) [reply .gitattributes]: Fixed up the comments in latest commit

File: .gitattributes (hunk 2 above)
Comment (ysims) [reply .gitattributes]: text=auto does this

Comment (JosephusPaye) [reply .gitattributes]: True 😅

**Pull request merged**